devbot/tools/api_requests.py

```diff
@@ -17,6 +19,7 @@
             json = await r.json()
             return json
         else:
+            LOGGER.error(f"Status code {r.status} returned from {r.url}")
             raise APIAccessError(f"HTTP status code {r.status} was returned")
```

Comment:
I think it would be more useful to let it return none if it can't access a website, instead of raising an error. 
With the current setup you'd probably wan't to catch this error anyways.

Reply:
Yes, but this way what is returned to the user is decided by the command author instead of me. Apart from that if the API link is wrong this will disable the command.